### ISSUE TITLE
Add NEON, SCAN - TPT, and SEINet external links to Arctos as an Ecosystem page

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#2599]
The MSB  is the external repository for legacy NEON mammal sampling (20,531
catalog records; 42,976 cryovials) from 2014–2018 (see [NEON portal](https://biorepo.neonscience.org/portal/collections/index.php)).

Arctos collections (MSB and HWML) are partners in the multi-institutional [The Parasite Tracker](http://parasitetracker.org/), an NSF-funded initiative to  digitizate and integrate of arthropod ectoparasite collections and their associated data with national (i.e., iDigBio) and global (i.e., GBIF) initiatives.
https://scan-bugs.org/portal/collections/misc/collprofiles.php?collid=235

@Jegelewicz @mvzhuang SEINet link for UTEP?

**Comment by Jegelewicz** [comment]
What do you need?

**Comment by Jegelewicz** [comment]
Here is the UTEP SCAN page https://scan-bugs.org/portal/collections/misc/collprofiles.php?collid=211

**Comment by Jegelewicz** [comment]
OK, I figured it out - here is UTEP SEINet http://swbiodiversity.org/seinet/collections/misc/collprofiles.php?collid=354

**Comment by campmlc** [comment]
Thanks, @Jegelewicz !
@mkoo  can we get these added to arctosdb.org Ecosystem links? We cite in proposal, should also make publicly available.

**Comment by campmlc** [comment]
Text to add? @Jegelewicz Please edit. Does UAM herbarium publish to a Symbiota portal?
@KyndallH 

Arctos collections include external repositories for legacy NEON sampling - samples are searchable through the NEON portal (https://biorepo.neonscience.org/portal/collections/index.php) and through the following Arctos project: [National Ecological Observatory Network (NEON) MSB Mammal BioArchive](https://arctos.database.museum/project/10002483)

Arthropod data are published by Arctos collections to Symbiota portals through SCAN ( https://scan-bugs.org/portal/collections/misc/collprofiles.php?collid=211) and The Parasite Tracker, an NSF-funded initiative to digitizate and integrate of arthropod ectoparasite collections and their associated data with national (i.e., iDigBio) and global (i.e., GBIF) initiatives.
https://scan-bugs.org/portal/collections/misc/collprofiles.php?collid=235

Arctos herbarium records are also served through the Symbiota SEINet portal at http://swbiodiversity.org/seinet/collections/misc/collprofiles.php?collid=354.

**Comment by dustymc** [comment]
I'm failing to see how these are part of the ecosystem. Repackaging some Arctos data as DWC just doesn't seem noteworthy to me. What's of value in those data? Is it somehow possible to even find records there?? Where's the "whoa, THAT's cool!"?

**Comment by campmlc** [comment]
These are NSF-funded initiatives that publish our data and have agreed link
to us. We should reciprocate, and announce our participation and
collaboration. It makes us look good, and gives the Arctos portal a bit
more publicity. We certainly need it, since no one knows we exist.

**Comment by dustymc** [comment]
>publicity

That's certainly of value, but I'm still not sure it's part of the ecosystem unless it does something cool for us.

>We should reciprocate

How can I do that?

**Comment by campmlc** [comment]
Yes, we put links back from our site to theirs, since they link from their
site to us. Where else would we put this except in the ecosystem page? This
is the same as GGBN. In fact, NEON is pulling from the GGBN resource.

**Comment by Jegelewicz** [comment]
These links go to individual collections, so just saying "Arctos collections" doesn't seem appropriate. Instead how about:

**Symbiota Portal Contributions**
Arctos collections contribute data to several Symbiot portals.

[Symbiota Collections of Arthropods Network (SCAN)](https://scan-bugs.org)
[Museum of Southwestern Biology arthropod data]( https://scan-bugs.org/portal/collections/misc/collprofiles.php?collid=211)
[UTEP Biodiversity Collections Arthropods](https://scan-bugs.org/portal/collections/misc/collprofiles.php?collid=211)

The Parasite Tracker
[Museum of Southwestern Biology Parasites(https://scan-bugs.org/portal/collections/misc/collprofiles.php?collid=235)

[SEINet](http://swbiodiversity.org)
[UTEP Biodiversity Collections Herbarium](http://swbiodiversity.org/seinet/collections/misc/collprofiles.php?collid=354)

**Comment by campmlc** [comment]
That's better.

**Comment by Jegelewicz** [comment]
> Where else would we put this except in the ecosystem page?

A lot of portals are publishing our data. I think we need a new page - Arctos in the Biodiversity Data Community

The iDigBio stuff should move there as should all of this. We should infor about VertNet, GBIF and others who are using our data.

**Comment by campmlc** [comment]
Great idea. Any objections?

**Comment by dustymc** [comment]
>we put links back from our site to theirs

Hu??

>same as GGBN

    >Arctos and Global Genome Biodiversity Network (GGBN) have worked on a common data standard to facilitate data sharing on genomic resources.

>info about VertNet, GBIF and others who are using our data.

We get something back from all of those (at least in theory) - vertnet does all sorts of work for us, GBIF has potentially-useful data checkers, etc.

>iDigBio

They fund digitization of Arctos collections.


>Arctos collections **contribute data** to several Symbiot portals.

I don't have a problem with that wording. I suppose in general listing things that only take is fine, but we should be specific about that. If NEON et al give nothing back then we shouldn't pretend otherwise, if they do then we should give them credit for it. I'm just asking for information....

**Comment by Jegelewicz** [comment]
> We get something back from all of those (at least in theory) - vertnet does all sorts of work for us, GBIF has potentially-useful data checkers, etc.

So then they ARE part of the Ecosystem? I'm pretty sure NEON paid MSB for the work they did (even if they have now left MSB with a bunch of stuff to care for in perpetuity with zero funding). The funding that helped get those arthropod collections digitized was contingent upon publication of data to SCAN (written into the TCN or PEN grants - from ADBC program at NSF). The only one that hasn't given back is SEINet - I chose to publish UTEP data there because the Curator wanted it there along with all of his collaborator's data. (We should pursue publishing all applicable Arctos collections to these portals just because the constituent communities prefer them to iDigBio or GBIF.)

>     iDigBio
> 
> They fund digitization of Arctos collections.

I disagree with the language on our ecosystem page. ADBC (aka NSF) funded that stuff as far as I know - iDigBio does provide us some visibility through the use of their webinar offering, but as far as I am concerned they cause problems too (Aves=Avus).

**Comment by Jegelewicz** [comment]
This whole thing probably just needs its own page as above

**Arctos in the Biodiversity Data Community**

> Arctos collaborates with other resources aimed at aggregating biodiversity data, improving data quality, and making the data available in broader contexts. These third-party systems complement Arctos by making the data more discoverable, and are in no way competitors of, or alternatives to, Arctos. Data sharing is based on the Darwin Core standard (Wieczorek et al. 2012). Summary data are published as Darwin Core Archives using the Integrated Publishing Toolkit (IPT) to VertNet, Global Biodiversity Information Facility (GBIF), Integrated Digitized Biocollections (iDigBio), Biodiversity Information Serving our Nation (BISON), Berkeley Ecoinformatics Engine, and other portals. It is important to note that these DarwinCore-based systems are neither a “database” nor a content management system.

**Comment by campmlc** [comment]
We should distinguish that Arctos hosts primary data, and the other portals
are aggregators.

How about this?

*Arctos in the Biodiversity Data Community*

Arctos collaborates with other resources aimed at aggregating biodiversity
data, improving data quality, and making the data available in broader
contexts. These third-party systems complement Arctos by making a subset of
Arctos data more discoverable through a variety of portals targeted to
specific user communities. Data sharing is based on the Darwin Core
standard (Wieczorek et al. 2012). Summary data are published as Darwin Core
Archives using the Integrated Publishing Toolkit (IPT) to VertNet, Global
Biodiversity Information Facility (GBIF), Integrated Digitized
Biocollections (iDigBio), Biodiversity Information Serving our Nation
(BISON), Berkeley Ecoinformatics Engine, NEON, SCAN, SEINet, and other
portals. It is important to note that these DarwinCore-based systems are
neither a “database” nor a content management system and may make
alterations or annotations to original collections data published via the
IPT. For access to all current collections data in real time, with
additional data and search fields extending beyond DarwinCore, please see
the Arctos search portal at https://www.arctos.database.museum.


and include some version of this also:

*Symbiota Portal Contributions*
Arctos collections contribute data to several Symbiota portals.

Symbiota Collections of Arthropods Network (SCAN) <https://scan-bugs.org/>
Museum of Southwestern Biology arthropod data
<https://scan-bugs.org/portal/collections/misc/collprofiles.php?collid=211>
UTEP Biodiversity Collections Arthropods
<https://scan-bugs.org/portal/collections/misc/collprofiles.php?collid=211>

The Parasite Tracker
[Museum of Southwestern Biology Parasites(
https://scan-bugs.org/portal/collections/misc/collprofiles.php?collid=235)

SEINet <http://swbiodiversity.org/>
UTEP Biodiversity Collections Herbarium
<http://swbiodiversity.org/seinet/collections/misc/collprofiles.php?collid=354>

**Comment by campmlc** [comment]
I guess we can't say this about Symbiota?
neither a “database” nor a content management system
They do provide tools for their users to enter and manage data directly.

**Comment by Jegelewicz** [comment]
As far as I can tell, none of this has ever been added. Do we care? @campmlc @mkoo or can we close this?

**Comment by campmlc** [comment]
Globi is part of our extended ecosystem, we feed to Neon via GGBN, this is
the broader connectivity and impact. There should be some place to add
these resources that are interconnected with ours.
